hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -413,67 +413,107 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		hcluster.Status.Version = computeClusterVersionStatus(r.Clock, hcluster, hcp)
 	}
 
-	// Set the ClusterVersionSucceeding based on the hostedcontrolplane
+	// Copy the CVO conditions from the HCP.
+	hcpCVOConditions := map[hyperv1.ConditionType]*metav1.Condition{
+		hyperv1.HostedControlPlaneDegraded:    meta.FindStatusCondition(hcp.Status.Conditions, string(hyperv1.HostedControlPlaneDegraded)),
```

Comment:
Should this be hyperv1.HostedControlPlaneFailing ? (looks like you already copy degraded below)